### PR TITLE
grid to flex on gamepage

### LINKED ISSUE #90: fix game page interface (desktop)
remove grid and try to build page using flex box

### PR DESCRIPTION
**Pull request by pdybowski**

### CODE REVIEW

**Review by bartoszbialecki — CHANGES_REQUESTED** [review]
Please combine with current master changes.

File: src/app/components/gameSettingsModal.ts [hunk 1]
```diff
@@ -116,15 +116,18 @@ export class GameSettingsModal {
...
     })
     const volumeParagraph = createElement({
       type: 'p',
       classes: ['volume__paragraph'],
+      styles: {
+        color: 'red',
+      },
       content: `
-        If you want to listean to background music, you need to enable audio in the site preferences!
+        (If you want to listean to background music, you need to enable audio in the site preferences!)
```
Comment (bartoszbialecki) [inline src/app/components/gameSettingsModal.ts]: typo: listen

File: src/assets/css/components/units.css [hunk 2]
```diff
@@ -6,6 +6,7 @@
   justify-content: center;
   align-items: center;
   width: 100%;
+  height: 100%:
```
Comment (bartoszbialecki) [inline src/assets/css/components/units.css]: should be semicolon

**Comment by hoolek77** [comment]
In my opinion game page should have a max-height of 100vh, but we can discuss it in Monday

**Pull request closed without merging**